joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/parser/RubyJsonHelpers.scala

```diff
@@ -169,7 +170,8 @@ object RubyJsonHelpers {
 
   /** Lowers the `||=` and `&&=` assignment operators to the respective `.nil?` checks
     */
-  def lowerAssignmentOperator(lhs: RubyExpression, rhs: RubyExpression, op: String, span: TextSpan): RubyExpression = {
+  def lowerAssignmentOperator(lhs: RubyExpression, rhs: RubyExpression, op: String, span: TextSpan): RubyExpression &
```

Comment:
Could you move this code to the `astcreation` package? Maybe there could be a `AstLoweringHelper` trait that can extend `AstCreator`

Reply:
Yep

Reply:
done